src/index.html

```diff
@@ -10,5 +10,159 @@
+          </p>
+        </li>
+        <li>
+          <p>
+              Business Administration<br />
+              @BizzNizz.net<br />
+              2010 - 2013<br />
+              <br />
+              This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+        <li>
+          <p>
+              Finance Director<br />
+              @Computers & More LTD.<br />
+              2008 - 2012<br />
+              <br />
+              This is a description of your work experience. Concisely describe your previous position in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+      </ul>
+    </span>
+    <div><h2>EDUCATION</h2></div>
+    <span>
+      <ul>
+        <li>
+          <p>
+              The San Francisco Business School<br />
+              2010 - 2012<br />
+              ​<br />
+              This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+        <li>
+          <p>
+              NYC School of Marketing and Advertising<br />
+              2007 - 2010<br />
+              <br />
+              This is a description of your education. Concisely describe your certifiate, degree or course in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+        <li>
+          <p>
+              L.A. University of Applied Sciences<br />
+              2002 - 2006<br />
+              ​<br />
+              This is a description of your education. Concisely describe your certificate, degree or course in a way that’s easy for readers to scan quickly.
+          </p>
+        </li>
+      </ul>
+    </span>
+    <div><h2>HOBBIES & INTERESTS</h2></div>
+    <span>
+      <ul>
+        <li>
+          <button><img src="Cinema.jpg" alt="Cinema" /></button>
+        </li>
+        <li>
+            <button><img src="Sports.jpg" alt="Sports" /></button>
+        </li>
+        <li>
+            <button><img src="Theatre.jpg" alt="Theatre" /></button>
+        </li>
+        <li>
+            <button><img src="BBQ.jpg" alt="BBQ" /></button>
+        </li>
+        <li>
+            <button><img src="Party.jpg" alt="Party" /></button>
+        </li>
+        <li>
+            <button><img src="Cooking.jpg" alt="Cooking" /></button>
+        </li>
+      </ul>
+    </span>
+    <div><h2>SKILLS</h2></div>
+    <span>
+      <ul>
+        <li>Microsoft Excel<br />Advanced</li>
+        <li>Microsoft Word<br />Advanced</li>
+        <li>Adobe<br />Proficient</li>
+        <li>Spanish<br />Advanced</li>
+        <li>PowerPoint<br />Proficient</li>
+        <li>French<br />Native</li>
+      </ul>
+    </span>
+    <span>
+      <h2>QUICK ID</h2><br />
```

Comment:
h2 is already a block element and the following content will begin from the next line. So the `<br />` is kinda redundant here.